config/unix-cuda.cmake

```diff
@@ -7,6 +7,10 @@
 
 include_guard(GLOBAL)
 
+if(NOT CMAKE_CUDA_COMPILER_ID STREQUAL NVIDIA)
```

Comment:
I think this is redundant since it's checked before the `unix-cuda.cmake` file is included. From `compilerEnv.cmake`:

```
781     # setup flags                                                                
782     if( "${CMAKE_CUDA_COMPILER_ID}" MATCHES "NVIDIA" )                           
783       include( unix-cuda )                                                       
784     else()                                                                       
785       message(FATAL_ERROR "Build system does not support "                       
786         "CUDACXX=${CMAKE_CUDA_COMPILER}")                                        
787     endif()                                                                      
788                          
```